src_startup/init_params_f92.asm

```diff
@@ -152,14 +152,33 @@ __init:			ld a, %FF
 ;
 			ld sp, __stack
 ;
-; Detect warm or cold boot
+; Detect warm or cold boot.
 ;
-			ld a, i			; Register I should be 0 on cold boot
-			or a, a		
+			ld de, __warmboot_magic	; If warm boot, __warmboot_magic (pushed by 'call _main') will

```

Comment:
I'm curious as to whether this cold boot detection can differentiate between a genuine cold power-on, or a press of the reset button

I find it valuable to be able to press the reset button, and then be able to do `OLD` inside BASIC, which is only possible because the memory hasn't been wiped

Reply:
verified that the latest changes now support a proper cold boot vs a reset-button press 👍